CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Identity
 ### Card scanning
 * [FIXED] [4548](https://github.com/stripe/stripe-android/pull/4548) Potential work leak when canceling a card scan in StripeCardScan
+* [ADDED] [4562](https://github.com/stripe/stripe-android/pull/4562) Add an example page for cardscan
```

Comment:
Since this touches change log, please also update it in the PR description's change log section